packages/client/test/unit/OrderingUtil.test.ts

```diff
@@ -31,53 +42,60 @@ describe('OrderingUtil', () => {
             assert.equal(to.timestamp, 3)
             assert.equal(to.sequenceNumber, 0)
             assert.equal(publisherId, 'publisherId')
-            done()
+            done.resolve(undefined)
         }
         util = new OrderingUtil( () => {}, gapHandler, 50, 50)
         const msg1 = msg
-        const msg4 = createMsg(4, undefined, 3)
+        const msg4 = await createMsg(4, undefined, 3)
         util.add(msg1)
         util.add(msg4)
+        await done
     })
-    it('does not call gap handler if gap detected but resolved before request should be sent', (done) => {
+
+    it('does not call gap handler if gap detected but resolved before request should be sent', async () => {
+        const done = Defer()
         const gapHandler = () => {
             throw new Error('The gap handler should not be called.')
         }
         util = new OrderingUtil(() => {}, gapHandler, 5000, 5000)
         const msg1 = msg
-        const msg2 = createMsg(2, undefined, 1)
-        const msg3 = createMsg(3, undefined, 2)
-        const msg4 = createMsg(4, undefined, 3)
+        const msg2 = await createMsg(2, undefined, 1)
+        const msg3 = await createMsg(3, undefined, 2)
+        const msg4 = await createMsg(4, undefined, 3)
         util.add(msg1)
         util.add(msg4)
         setTimeout(() => {
             util.add(msg2)
             util.add(msg3)
-            done()
+            done.resolve(undefined)
         }, 500)
+        await done
     })
-    it('handles unordered messages in order (large randomized test)', () => {
-        const msg1Pub1 = createMsg(1, 0, null, 0, {}, 'publisherId1')
-        const msg1Pub2 = createMsg(1, 0, null, 0, {}, 'publisherId2')
-        const msg1Pub3 = createMsg(1, 0, null, 0, {}, 'publisherId3')
+
+    it('handles unordered messages in order (large randomized test)', async () => {
+        const msg1Pub1 = await createMsg(1, 0, null, 0, {}, 'publisherId1')
+        const msg1Pub2 = await createMsg(1, 0, null, 0, {}, 'publisherId2')
+        const msg1Pub3 = await createMsg(1, 0, null, 0, {}, 'publisherId3')
         const expected1 = [msg1Pub1]
         const expected2 = [msg1Pub2]
         const expected3 = [msg1Pub3]
-        for (let i = 2; i <= 100000; i++) {
-            expected1.push(createMsg(i, 0, i - 1, 0, {}, 'publisherId1'))
+        for (let i = 2; i <= 100; i++) {
+            expected1.push(await createMsg(i, 0, i - 1, 0, {}, 'publisherId1'))
         }
-        for (let i = 2; i <= 100000; i++) {
-            expected2.push(createMsg(i, 0, i - 1, 0, {}, 'publisherId2'))
+        for (let i = 2; i <= 100; i++) {
+            expected2.push(await createMsg(i, 0, i - 1, 0, {}, 'publisherId2'))
         }
-        for (let i = 2; i <= 100000; i++) {
-            expected3.push(createMsg(i, 0, i - 1, 0, {}, 'publisherId3'))
+        for (let i = 2; i <= 100; i++) {
+            expected3.push(await createMsg(i, 0, i - 1, 0, {}, 'publisherId3'))
```

Comment:
Maybe total 30k?

Actually maybe we can just use fake signatures and keep 300k?